Rocket.Chat/Controllers/Chat/ChatViewControllerBanner.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ChatViewControllerBanner.swift
+//  Rocket.Chat
+//
+//  Created by Matheus Cardoso on 7/5/18.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import UIKit
+
+extension ChatViewController {
+    func showBanner(_ model: ChatBannerViewModel) {
+        bannerView?.model = model
+        bannerView?.isHidden = false
+    }
+
+    func hideBanner() {
+        bannerView?.isHidden = true
+    }
+
+    func setupBanner() -> ChatBannerView? {
+        guard let bannerView = ChatBannerView.instantiateFromNib() else {
+            return nil
+        }
+
+        bannerView.translatesAutoresizingMaskIntoConstraints = false
+
+        view.addSubview(bannerView)
+
+        view.addConstraints(
+            NSLayoutConstraint.constraints(
```

Comment:
What do you think about using layout anchors here?

Reply:
Noted :+1: I'll change it soon.